data/json/recipes/recipe_others.json

```
@@ -3380,5 +3380,23 @@
     "qualities": [ { "id": "CUT", "level": 2 } ],
     "components": [ [ [ "stick", 1 ] ], [ [ "sheet_canvas", 1 ], [ "sheet_canvas_patchwork", 1 ] ] ],
     "byproducts": [ [ "splinter", 8 ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "hobo_candle",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "difficulty": 3,
+    "time": "10 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 2 } ],
+    "components": [
+      [ [ "gum", 3 ], [ "super_glue", 10 ], [ "bone_glue", 1 ] ],
+      [ [ "jar_3l_glass_sealed", 1 ] ],
+      [ [ "string_36", 1 ] ],
```

Does this really need 90cm string as a wick? I suppose it's a bit of an overkill.
Also I suggest it could use cotton patchwork as well.
```suggestion
      [ [ "string_6", 1 ], [ "cotton_patchwork", 1 ] ],
```

---

I fully agree with the cotton patchwork, as alcohol lanterns use basically a thick cotton patch rolled or woven into a wick.
However the long string was included for 3 reasons:
- You get a long string from tearing down curtains, giving less frinction for new players getting through the early game who might not yet realize you can just cut it up
- The string will need to be folded and twisted a few times, to promote fumes comming out, since the point is that the fumes mostly burn, not just the wick. I'm not sure how many, but at least once if not more.
- The candle doesn't degrade, even though the wick would burn through at least a little. This lenght should give us some wiggle room, with the survivor basically pulling the wick out a little every time it gets too short. Although this may be remedied by needing to replace the wick once in a while, I'm not sure how (if even possible) to do that.

---

Ok, got it.